index.html

```diff
@@ -20,80 +20,61 @@
     <title>Victor's Portfolio Page</title>
 </head>
-<body>
-	
-  <header class="menu-bar">
-    <a  class="img-logo" href="#">my logo</a>
-    <img src="img/Union.svg" alt="Hamburger">
-    <ul class="desktop-menu">
-      <li><a href="#">portfolio</a></li>
-      <li><a href="#">about</a></li>
-      <li><a href="#">contact</a></li>
-    </ul>
-	</header>
-  <section class="about-myself-top">
-    <div class="intro">
-      <h1 class="animation-h">I’m Victor Ozoede Glad to see you!</h1>
-      <p class=animation-p>I’m a software developer! I can help you build a product , feature or website Look through   some of my work and experience! If you like what you see and have a project you need coded, don’t   hestiate to contact me.</p>
-    </div>
-    <a class="connect connect-header" href="#">let's connect</a>
-    <ul class="social-icons icons-header">
-      <li><a href="#"><i class="icons fab fa-twitter"></i></a></li>
-      <li><a href="www.linkedin.com/in/chukwuebuka-ozoede-46616a219"><i class="icons fab fa-linkedin-in"></i></a></li>
-      <li><a href="#"><i class="icons fab fa-medium-m"></i></a></li>
-      <li><a href="https://github.com/VictorChukwuebuka"><i class="icons fab fa-github"></i></a></li>
-      <li><a href="#"><i class="icons fab fa-angellist"></i></a></li>
-		</ul>  
-  </section> 
-  
-  <main class="grid">
-    <!-- First section starts -->
-    <section class="work-one work">
-      <img class ="img-work img-work-one" src="img/Portfolio1.svg" alt="Facebook 360">
-        <div class="body-content">
-        <h2>Facebook 360</h2>
-          <ul class="work-top">
-            <li class="canopy">CANOPY</li>
-            <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
-            <li class="dev">back end dev</li>
-            <li><img class="counter" src="img/Counter.svg" alt="counter"></li>
-            <li class="dev">2015</li>
-          </ul>
-          <p class="body-paragraph">
-            A daily selection of privately personalized reads; no accounts or sign-ups required.
-          </p>
-          <ul class="work-bottom work-bottom1">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
-          </ul>
-          <ul class="work-bottom2">
-            <li>html</li>
-            <li>Ruby on rails</li>
-            <li>css</li>
-            <li>javascript</li>
-          </ul>
-          <form action="#">
-              <button class="btn-body btn" type="submit" name="submit">see project</button>
-          </form>
+  <body>
+    <header class="menu-bar">
+      <a class="img-logo" href="#">my logo</a>
+      <ul class="desktop-menu">
+        <li><a href="#">portfolio</a></li>
+        <li><a href="#">about</a></li>
+        <li><a href="#">contact</a></li>
+      </ul>
+      <button class="ham" type="button">
+        <span class="menuIcon">
+          <img src="../img/menuIcon.svg" alt="menu-icon">
```

Comment:
- [ ] The hamburger menu button is not displaying correctly. kindly try to fix it.

![image](https://user-images.githubusercontent.com/53019916/131435898-e9e74ca5-ff14-4813-b47a-7efc31b411a8.png)